pkg/common/krill-ubuntu-jammy.krill.service

```diff
@@ -0,0 +1 @@
+krill-ubuntu-focal.krill.service
```

Comment:
This looks like the file content is missing - it just has the name of the file?

Reply:
It's a sym link.

Reply:
oh right.. sorry I was confused about the way this is represented in GH